src/Scene/LayersConfiguration.js

```diff
@@ -4,6 +4,7 @@
  * Description: Le layer est une couche de données. Cette couche peut etre des images ou de l'information 3D. Les requètes de cette couche sont acheminées par une interfaceCommander.
  *
  */
+import { l_COLOR, l_ELEVATION } from 'Renderer/LayeredMaterial';
```

Comment:
I'd rather not do this.
If you really need to be able to do `layer.type = l_ELEVATION;` then the `l_ELEVATION` constant probably belong to this file.
But I'd rather not have this at all - and it's unrelated to the current PR scope anyway ?